test/expect/HasDecompTest.test_has_decomposition.expect

```diff
@@ -508,6 +508,8 @@ aten::_transformer_encoder_layer_fwd
 aten::_transformer_encoder_layer_fwd.out
 aten::_trilinear
 aten::_trilinear.out
+aten::_triton_bsr_dense_mm
+aten::_triton_bsr_dense_mm.out
```

Comment:
@cpuhrsch , this is very weird. I have just reordered triton kernels with the ours at the very top, and the issues are miraculously gone.